stdlib/public/Concurrency/DistributedActor.swift

```diff
@@ -0,0 +1,186 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2020 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Swift
+@_implementationOnly import _SwiftConcurrencyShims
+
```

Comment:
That's the new types we're adding @DougGregor 

They're not _really_ "Concurrency" but rather "Distributed", but at the same time all checking, conforming to and synthesis of those is done very close to concurrency since it is all actor related.

Should we introduce a new directory/package/module like `_Concurrency` for `Distributed` and put those types there?